tests/Doxyfile

```diff
@@ -0,0 +1,1854 @@
+# Doxyfile 1.8.3.1
```

Comment:
Why's this all in tests, or is it a limitation of the build and not polluting the uC codespace?